README.md

```diff
@@ -1,6 +1,20 @@
 # petclinic-plugins
 
->From petclinic-plugins root:
-* Publish artifacts to maven local: `gradle clean publishToMavenLocal`
-* For using idea plugin add - apply plugin: 'petclinic-idea', it applies configurations for idea for
--checkstyle, formatter, pmd, findbugs plugins(BUT in addition: CheckStyle-IDEA, PMDPlugin, FindBugs-IDEA should be installed on ide)
+### How to use:
+* From petclinic-plugins root publish plugins to maven local repository: `gradle clean publishToMavenLocal`
+* For using it from child project add to buildscript dependencies in build.gradle:
+    `classpath("com.epam.petclinic:petclinic-plugins:${petclinicPluginsVersion}")`
+
+### Available plugins:
+Add to build.gradle
+* `apply plugin: 'com.epam.petclinic.java'` - applies checkstyle, formatter, pmd, findbugs code quality checkers for gradlew build
+* `apply plugin: 'com.epam.petclinic.idea'` - applies idea configurations for checkstyle, formatter, pmd, findbugs plugins(BUT in addition: CheckStyle-IDEA, PMDPlugin, FindBugs-IDEA should be installed on ide)
```

Comment:
Nothing, you could install it and its configuration fetched after idea is restarted